server/routes/companies.js

```diff
@@ -2,14 +2,174 @@
 var express = require('express');
 var router = express.Router();
 
+const db = require('../db')
+
 // Returns test string to verify that Companies server is running.
 router.get('/ping', function(req, res, next) {
   res.send('pong - Companies API');
 });
 
-// GET companies listing.
+//Get all companies hiring a specific major
+//written before /: endpoints since those can have various names
```

Comment:
Another general note: I know there were some issues where /:company_id was matching with other routes such as /filter, etc. but we should maintain the order of routes to match that in the documentation for readability and convenience.

Also, any changes to the routes should be updated in the documentation (e.g. changing the route /:company_id to /:company_id/id to prevent conflicts with other routes)

Reply:
I reordered everything to match to original week one document / documentation (resultingly, filter and search cause errors when they are called). Let me know if you want me to change the order / documentation

Reply:
Sorry I was a bit unclear, the "(e.g. changing the route /:company_id to /:company_id/id to prevent conflicts with other routes)" means change that route to /:company_id/id so there won't be errors with filter and search

Reply:
Just fixed it. I also updated the documentation